modules/ROOT/pages/alerts-on-runtime-manager.adoc

```diff
@@ -1,317 +1,460 @@
+* Application undeployed
+* Deployment failure
+* Deployment success
 
 
-* *CPU Usage* - CPU usage of the servers.
-+
-image::cpu-usage.png[cpu]
-+
-To enable this condition, specify values in the duration and capacity threshold parameters that, when crossed, trigger the alert.
+=== Conditions for Local Servers
 
+image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
+The available alert events depend on the server type.
+The following alerts are available for all target types (servers and nodes in server groups and clusters):
 
-* *Memory Usage* - Total memory usage in MB on the server.
-+
-image::memory-usage.png[memo]
+
+Server CPU usage::
 +
-To enable this condition, specify values in the duration and capacity threshold parameters that, when crossed, trigger the alert.
+--
+The CPU usage of the servers that run the application reaches a capacity threshold in a specified period.
 
-* *Server Load Average* - Average processing for a given time period.
+image::alerts-conditions-servers-cpu.png[Server CPU usage alert condition]
 
-+
-image::alerts-on-runtime-manager-load-average.png[]
+To enable this condition, specify the CPU usage threshold and duration that triggers the alert.
+--
 
+Server Load average::
 +
-[NOTE]
-Windows environments don’t provide this information, so this metric isn't available on Windows environments.
+--
+The average processing load of the server is above or below a threshold in a specified period.
+
+image::alerts-conditions-servers-load-avg.png[Server load average alert condition]
 
-* *Server Thread Count* - Number of threads on the JVM.
+To enable this condition, specify the load threshold and duration that triggers the alert.
 
+[NOTE]
+Windows environments don't provide this information, so this metric isn't available on Windows environments.
+--
+
+Server Memory usage::
 +
-image::alerts-on-runtime-manager-thread-count.png[]
+--
+The total memory usage (in MB) on the server is above or below a threshold in a specified period.
 
+image::alerts-conditions-servers-memory.png[Server memory usage alert condition]
 
+To enable this condition, specify the memory usage threshold and duration that triggers the alert.
+--
 
-[NOTE]
-None of the alerts listed above are currently available on the xref:deployment-strategies.adoc#deployment-scenarios[Anypoint Platform Private Cloud Edition].
+Server Thread count::
++
+--
+The thread count on the JVM is above or below a threshold for a specified period.
```

Comment:
Can a threshold include aboves AND belows? Is it a numerical span rather than just one number? And if just one number, why bother with a "threshold" when there is just one value the count can be?